[EN/ZH] Add `osu! FumoFumo Cup: Summer Lights` wiki article (#13679)
ppy/osu-wiki: wiki/Tournaments/OFFC/summer-lights/en.md

![OFFC: SL logo](img/logo.png)

The **osu! FumoFumo Cup: Summer Lights** (***OFFC: SL***) is a 4v4 double-elimination osu!standard tournament hosted by ::{ flag=CN }:: [- Isuzu Ren -](https://osu.ppy.sh/users/13217389), which incorporates a brand-new chess-style ban / pick gameplay mechanic based on the traditional tournament rules. It is the second season of the osu! FumoFumo Cup tournament series.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2025-06-15/2025-06-30 |
| Team-up | 2025-07-01 |
| Quarterfinals | 2025-07-01/2025-07-14 |
| Semifinals | 2025-07-16/2025-07-29 |
| Finals | 2025-08-01/2025-08-14 |
| Grand Finals | 2025-08-16/2025-08-22 |

## Prizes

### Banners

- All participants will get a customized profile banner and a title in the official QQ group.
- The top three teams will get special banners and titles.
- All staff will get a customized profile banner.
- All customized beatmap mappers will get a customized beatmap banner.

### Round Champions

All team members with the best overall performances in each round will receive a commemorative champion entity medal for the respective round.

### Special Prizes

> For MVP and SVP players, the MVP could choose one Fumo from **Komeiji Koishi Fumo** and **Flandre Scarlet Fumo**, and then the SVP would get the other one as his / her prize.

Reviewer: ```suggestion
The player whom the tournament management decides to be the tournament's "MVP" (Most Valuable Player) may choose to pick **a Fumo figurine** of either *Komeiji Koishi* or *Flandre Scarlet* as their special prize, with the player who ends up as the tournament's "SVP" (Second-Most Valuable Player) getting the other Fumo by default.
```

Reviewer: also might want to add a link the fumos here if you have the marketplace link

Author: Where to buy the Fumo is still undecided, and it's still unable to add the link for now 😅

> For the Best Planned and Best Labor Strategists, the Best Planned strategist could choose one set of Blue Archive character stand and badge (Summer Party) of *Kasumizawa Miyu* or *Ajitani Hifumi*. And the Best Labor strategist would get the other set.

Reviewer: ```suggestion
The player whom the public polls decide to be the tournament's "Best Planned Strategist" may choose to pick a **Blue Archive character stand and badge (Summer Party)** of either *Kasumizawa Miyu* or *Ajitani Hifumi* as their special prize, with the player who gets voted as the "Best Labor Strategist" getting the other set by default.
```

| Player | Prize |
| :-- | :-- |

> | MVP (from 1st place team) | One Fumo |

Reviewer: ```suggestion
| MVP (from the winning team) | One Fumo |
```

> | SVP (from 2nd place team) | One Fumo |

Reviewer: ```suggestion
| SVP (from the runner-up team) | One Fumo |
```

| Best Planned Strategist (from the polls) | Blue Archive character stand + badge |
| Best Labor Strategist (from the polls) | Blue Archive character stand + badge |
| Player with the highest overall HD score | だから僕は音楽を辞めた【初回限定盤】（ヨルシカ） CD music album |
| Player with the highest overall HR score | サマーライト（夏央、倚水） CD music album |

## Organization

The osu! FumoFumo Cup: Summer Lights is run by various community members.

| Position | Member(s) |
| :-- | :-- |
| Host | ::{ flag=CN }:: [- Isuzu Ren -](https://osu.ppy.sh/users/13217389) |

> | Rule maker | ::{ flag=CN }:: [- Isuzu Ren -](https://osu.ppy.sh/users/13217389), ::{ flag=CN }:: [kitakore](https://osu.ppy.sh/users/33887163), ::{ flag=CN }:: [NaughtyChas](https://osu.ppy.sh/users/15349720) |

Reviewer 2: isn't "rulemaker" a word though?

> | Rule maker | ::{ flag=CN }:: [- Isuzu Ren -](https://osu.ppy.sh/users/13217389), ::{ flag=CN }:: [kitakore](https://osu.ppy.sh/users/33887163), ::{ flag=CN }:: [NaughtyChas](https://osu.ppy.sh/users/15349720) |

Author: I saw both rulemaker and rule-maker on websites, unsure about which one is used more or correct.

| **Team H** | zako psc | ::{ flag=HK }:: [Fallen_HK](https://osu.ppy.sh/users/13981456), ::{ flag=CN }:: [eric1388](https://osu.ppy.sh/users/20384257), ::{ flag=HK }:: [Mithra Tercera](https://osu.ppy.sh/users/2955661), ::{ flag=CN }:: [RABBIT2](https://osu.ppy.sh/users/10260284), ::{ flag=MO }:: [7White](https://osu.ppy.sh/users/17159216), ::{ flag=TW }:: [Mou li kai](https://osu.ppy.sh/users/30593882), ::{ flag=CN }:: [Pouxba](https://osu.ppy.sh/users/16378561), ::{ flag=CN }:: [comgrat](https://osu.ppy.sh/users/12830046) |

## Podium

This competition has come to an end and resulted in the following podium:

| Placing | Team |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 多啦A梦 |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | zako psc |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 不是夏天我不草 |

> Round champion teams are as follows:

Reviewer: ```suggestion
The winning teams for each individual rounds are as follows:
```

| Round | Team |
| :-: | :-- |
| Quarterfinals | 都是SummerSky害的 |
| Semifinals | 给你一拳 |
| Finals | zako psc |
| Grand Finals | 多啦A梦 |

> Those players are awarded with corresponding prizes:

Reviewer: ```suggestion
These players are awarded with the corresponding special prizes:
```

> | Placing | Player |

Reviewer: ```suggestion
| Distinction | Player |
```

- Tiebreaker: [Xyris - Beyond the Edge (feat. Hanakuma Chifuyu) [Limitless]](https://osu.ppy.sh/beatmapsets/2410026#osu/5232626)

## Match Results

### Quarterfinals

Match time: 5 July 2025 ~ 7 July 2025

| Team 1 | Team 2 | Match link |
| --: | :-- | :-- |
| **不是夏天我不草** | 原來OFFC:SL的S不是suiji的意思嗎 | [#1](https://osu.ppy.sh/community/matches/118591736) |
| 让你选DT你龙吗 | **多啦A梦** | [#2](https://osu.ppy.sh/community/matches/118591049) |
| Valorant after this | **zako psc** | [#3](https://osu.ppy.sh/community/matches/118591753) |
| 给你一拳 | **都是SummerSky害的** | [#4](https://osu.ppy.sh/community/matches/118610985) |

### Semifinals

Match time: 20 July 2025 ~ 27 July 2025

| Team 1 | Team 2 | Match link |
| --: | :-- | :-- |
| **不是夏天我不草** | 都是SummerSky害的 | [#5](https://osu.ppy.sh/community/matches/118730743) |
| zako psc | **多啦A梦** | [#6](https://osu.ppy.sh/community/matches/118787262) |
| **给你一拳** | 原來OFFC:SL的S不是suiji的意思嗎 | [#7](https://osu.ppy.sh/community/matches/118797298) |
| 让你选DT你龙吗 | **Valorant after this** | [#8](https://osu.ppy.sh/community/matches/118796796) |

### Finals

Match time: 10 Argust 2025 ~ 11 Argust 2025

| Team 1 | Team 2 | Match link |
| --: | :-- | :-- |

> | 不是夏天我不草 (Forfeit) | **多啦A梦** | - |

Reviewer: ```suggestion
| 不是夏天我不草 | **多啦A梦** | N/A (*win by default*) |
```
